addons/l10n_kh_pos/static/src/overrides/components/order_table_line/order_table_line.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="l10n_kh_pos.OrderTableLine">
+        <t t-set="line" t-value="props.line" />
+        <tr class="lh-sm">
+            <td class="py-2 align-top">
+                <div t-attf-class="{{ line.comboParent ? 'border-start border-3 ms-2 ps-2' : '' }}">
+                    <div class="text-wrap fw-bolder" t-out="line.productName"/>
+                    <div t-if="line.unit">
+                        kh_unit / Unit: <t t-out="line.unit"/>
```

Comment:
kh_unit?
I suppose it's missing translations?

Reply:
yes it should be the Khmer for unit
but there is still no conclusion on whether these fields should be at all, so it's left like this